README.md

```diff
@@ -1,13 +1,25 @@
 ## Doppler
-
 [![CircleCI](https://circleci.com/gh/artsy/doppler.svg?style=svg)](https://circleci.com/gh/artsy/doppler)
 [![Code Climate](https://codeclimate.com/github/artsy/doppler.svg)](https://codeclimate.com/github/artsy/doppler)
 
-I am Doppler, the Artsy developer website. 
+I am Doppler, the Artsy developer website.
 
 - **Production:** [developers.artsy.net](https://developers.artsy.net)
 - **Staging:** [developers-staging.artsy.net/](https://developers-staging.artsy.net/)
 
+---
+
+### Public API Retirement Notice
+Please note, we are in the process of retiring the [public api.](developers.artsy.net/v2).
+
+The public api, as well as its documentation and playground, will remain available until **June 30th, 2025.**
```

Comment:
Somewhat arbitrary date, but arguably gives enough time to us and those who will receive a notice about its retirement.